core/src/test/scala/unit/kafka/log/LogCleanerTest.scala

```diff
@@ -2070,18 +2071,55 @@ class LogCleanerTest extends Logging {
     cleaner3.lastStats.bufferUtilization = 0.65
     cleaners += cleaner3
 
-    assertEquals(0, logCleaner.maxOverCleanerThreads(_.lastStats.bufferUtilization))
+    assertEquals(0.85, logCleaner.maxOverCleanerThreads(_.lastStats.bufferUtilization))
+  }
+
+  @Test
+  def testMaxBufferUtilizationPercentMetric(): Unit = {
```

Comment:
It seems this new unit test is similar to `testMaxOverCleanerThreads`, right? If so, could you please remove `testMaxOverCleanerThreads`?